app/src/layouts/DashboardLayout/index.jsx

```diff
@@ -74,10 +74,11 @@ const DashboardLayout = () => {
         <div className="app-main-content">
           <DashboardContent />
         </div>
-
-        <div className="app-footer">
-          <Footer />
-        </div>
+        {!isAppTypeSessionBear() ? (
```

Comment:
create a memoised `isFooterVisible` (_similar to `isSideBarVisible`_) and use that. This will keep the markup clean.